mlkem/native/api.h

```diff
@@ -249,7 +319,15 @@ static INLINE void poly_frombytes_native(int16_t a[MLKEM_N],
  * len).
  **************************************************/
 static INLINE int rej_uniform_native(int16_t *r, unsigned int len,
-                                     const uint8_t *buf, unsigned int buflen);
+                                     const uint8_t *buf, unsigned int buflen)
```

Comment:
Tiny nitpick: did we settle on using "unsigned int" or just "unsigned" consistently? In the main mlkem sources, we appear to be using just "unsigned", while the FIPS202 sources contain "unsigned int". Do we care?

Reply:
Good spot, we should clean that up at some point. Do you mind opening an issue?

Reply:
OK.

Reply:
Opened https://github.com/pq-code-package/mlkem-native/issues/689

Reply:
Thanks!